config/webpack.config.js

```diff
@@ -227,7 +214,7 @@ const configGenerator = (options, apps) => {
     baseConfig.devtool = '#eval-source-map';
   }
 
-  if (options.analyzer) {
+  if (buildOptions.analyzer) {
     baseConfig.plugins.push(
       new BundleAnalyzerPlugin({
         analyzerMode: 'disabled',
```

Comment:
There were several variations of the same condition here that I ultimately consolidated into `isOptimizedBuild`.